### PR TITLE
[INLONG-579][Doc] Add doc for oracle connector for all migrate

### LINKED ISSUE #579: [Feature][Doc] Add doc for oracle connector for all migrate
### Description

Add documents to describe how to use the multiple database migration of oracle connector.

### Use case

see docs.

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by e-mhui**
### Add doc for oracle connector for all migrate

- Fixes #579

### Motivation

Add doc for oracle connector for all migrate

### CODE REVIEW

File: docs/data_node/extract_node/oracle-cdc.md [hunk 1]
```diff
@@ -328,6 +328,13 @@ TODO: It will be supported in the future.
       <td>String</td>
       <td>Inlong metric label, format of value is groupId&streamId&nodeId.</td> 
     </tr>
+    <tr>
+       <td>source.multiple.enable</td>
+       <td>optional</td>
+       <td style={{wordWrap: 'break-word'}}>false</td>
+       <td>Boolean</td>
+       <td>Whether to enable multiple schema and table migration. If it is' true ', Oracle Extract Node will compress the physical field of the table into a special meta field' data 'in the format of 'canal json'.</td>
```
Comment (EMsnap) [inline docs/data_node/extract_node/oracle-cdc.md:336]: data -> data_canal or data_debezium, data is deprecated

Comment (EMsnap) [inline docs/data_node/extract_node/oracle-cdc.md:336]: data -> data_canal or data_debezium, data is deprecated

**Review by yunqingmoswu — COMMENTED** [review]
Is it better to describe the Oracle whole database migration?

File: docs/data_node/extract_node/oracle-cdc.md (hunk 1 above)
Comment (e-mhui) [reply docs/data_node/extract_node/oracle-cdc.md]: done.

**Review by EMsnap — APPROVED** [review]

**Review by gong — APPROVED** [review]

File: docs/data_node/extract_node/oracle-cdc.md [hunk 2]
```diff
@@ -427,6 +500,36 @@ _Note: the mechanism of `scan.startup.mode` option relying on Debezium's `snapsh
 
 The Oracle Extract Node can't work in parallel reading, because there is only one task can receive change events.
 
+### Whole Database, Multiple Schemas, Multiple Tables Migration
```
Comment (yunqingmoswu) [inline docs/data_node/extract_node/oracle-cdc.md:503]: It is recommend to add a title for it such as '##Features'.

Comment (e-mhui) [reply docs/data_node/extract_node/oracle-cdc.md:503]: Its upper level is a '## Features'

**Review by yunqingmoswu — APPROVED** [review]

**Pull request merged**